waypoints-special/CAN_PG_A-M.cup

```diff
@@ -1,415 +1,414 @@
-"Mundare",Mundare,CA,5335.700N,11220.167W,688.8m,1,,,,"Turnpoint."
-"Parkland",Parkland,CA,5015.366N,11339.376W,1063.8m,1,,,,"Turnpoint."
-"Parkview",Parkview,CA,4910.552N,11349.896W,1368.9m,1,,,,"Turnpoint."
-"Pendleton",CNF3,CA,4529.166N,07505.766W,79.2m,2,260,777.2m,"123.3","ATF 5 nm 3300 asl 08/26 Land on Grass SOUTH 13/31 Land on Grass NORTH 17/35 Land on Grass WEST ALL RWYs 2550 x 100 DO NOT LAND ON PAVEMENT"
-"Pincher Crk",PinchCrk,CA,4929.742N,11356.887W,1144.8m,1,,,,"Turnpoint."
+"Mundare",Mundare,CA,5335.700N,11220.167W,688.8m,1,,,,
+"Parkland",Parkland,CA,5015.366N,11339.376W,1063.8m,1,,,,
+"Parkview",Parkview,CA,4910.552N,11349.896W,1368.9m,1,,,,
+"Pendleton",CNF3,CA,4529.166N,07505.766W,79.2m,2,260,777.2m,"123.300","ATF 5 nm 3300 asl 08/26 Land on Grass SOUTH 13/31 Land on Grass NORTH 17/35 Land on Grass WEST ALL RWYs 2550 x 100 DO NOT LAND ON PAVEMENT"
+"Pincher Crk",PinchCrk,CA,4929.742N,11356.887W,1144.8m,1,,,,
 "Pine Lake Northw",PINELA,CA,5206.700N,11328.500W,903.0m,1,,,,
 "St Dominique City",STDO_CTY,CA,4533.883N,07251.150W,89.9m,1,,,,
 "St Hyacinthe",STHYAIRP,CA,4536.300N,07300.850W,36.0m,5,20,1170.1m,"123.200","CSU3 5NM 3200 ASL"
 "St Liboire",STLIBOIR,CA,4539.067N,07245.750W,97.5m,1,,,,
 "St Michael",StMichael,CA,5349.644N,11237.422W,649.2m,1,,,,
 "St Valerien",STVALERI,CA,4533.917N,07242.583W,76.0m,1,,,,
-"Standoff",Standoff,CA,4928.767N,11318.067W,986.0m,1,,,,"Turnpoint."
+"Standoff",Standoff,CA,4928.767N,11318.067W,986.0m,1,,,,
 "Ste Helene",STHELENE,CA,4543.867N,07244.184W,75.0m,1,,,,
 "Ste Rosalie",STROSALI,CA,4538.767N,07253.933W,16.0m,1,,,,
 "Three Hills Ab",THREE,CA,5141.800N,11312.550W,908.0m,5,110,920.0m,"123.200",
 "Trans Canada",Transada,CA,4533.935N,07430.119W,50.9m,1,,,,
-"Twin Butte",TwButte,CA,4915.600N,11351.600W,1363.4m,1,,,,"Turnpoint, Highway 6 T-intersection to South."
+"Twin Butte",TwButte,CA,4915.600N,11351.600W,1363.4m,1,,,,Highway 6 T-intersection to South."
 "Two Hills",TwoHills,CA,5342.000N,11145.964W,596.0m,1,,,,
-"Vankleek Hill",VNKLKHLL,CA,4531.183N,07439.133W,115.8m,1,,,,"Turn Point, Rd. Int."
-"Vegreville",Vegrevil,CA,5330.867N,11201.650W,631.5m,5,130,1219.2m,"122.8","Turnpoint, (UNICOM o/t tfc).  Righthand circuits Rwy 31.  NOTE:  Gun club 1 nm NW.  Avoid or overfly no less than 1000 ft AGL.  Ultralight activity in vicinity.  13/31."
+"Vankleek Hill",VNKLKHLL,CA,4531.183N,07439.133W,115.8m,1,,,,
+"Vegreville",Vegrevil,CA,5330.867N,11201.650W,631.5m,5,130,1219.2m,"122.8",(UNICOM o/t tfc).  Righthand circuits Rwy 31.  NOTE:  Gun club 1 nm NW.  Avoid or overfly no less than 1000 ft AGL.  Ultralight activity in vicinity.  13/31."
 "Wimborne Hamlet",WINBOH,CA,5151.600N,11335.500W,946.0m,1,,,,
 "Woodstock",Woodsown,CA,4306.582N,08049.248W,295.7m,1,,,,
 -----Related Tasks-----
-"AVVC 1 - St Dominique","CSS4","St Dominique City","St Liboire","Ste Rosalie","St Dominique City","CSS4"
+"AVVC 1 - St Dominique","CSS4","St Dominique City","St Liboire","Ste Rosalie","St Dominique City","CSS4",,,,
```

Comment:
So here according to cup standard http://download.naviter.com/docs/CUP-file-format-description.pdf
The ,,,, are not necessary. Its just a task name and a list of turn points.